patches/chrome-browser-net-net_error_tab_helper.cc.patch

```diff
@@ -1,14 +1,12 @@
 diff --git a/chrome/browser/net/net_error_tab_helper.cc b/chrome/browser/net/net_error_tab_helper.cc
-index 313758372e0fcb842ee6a5b5cd69a64b8a95b2ac..cc920fb9459462075a1a9c123ae7af34ea5c8ac8 100644
+index 313758372e0fcb842ee6a5b5cd69a64b8a95b2ac..bc8c3c243dd2f05b5fa9708d1d72bfa9f6dffcc0 100644
 --- a/chrome/browser/net/net_error_tab_helper.cc
 +++ b/chrome/browser/net/net_error_tab_helper.cc
-@@ -200,6 +200,9 @@ void NetErrorTabHelper::InitializePref(WebContents* contents) {
+@@ -200,6 +200,7 @@ void NetErrorTabHelper::InitializePref(WebContents* contents) {
  }
  
  bool NetErrorTabHelper::ProbesAllowed() const {
-+#if defined(BRAVE_CHROMIUM_BUILD)
```

Comment:
maybe we could develop a way to avoid creation of certain tabhelpers, because it seems the whole thing is not needed for us

Reply:
I think I lack context to properly understand what you mean here. Could you please elaborate a bit more?

Reply:
It appears to me that the whole point of this class is sending some sort of "probes" which we disable in this method. So instead of patching we could maybe avoid instantiating the helper (it's unclear to me how exactly, just a raw idea). And probably there are more unneeded helpers (or keyed services) like this.

Reply:
I see what you mean now, thanks for the clarification.

Reply:
Handled in https://github.com/brave/brave-browser/issues/14219